en/software_update/stm32_bootloader.md

```diff
@@ -16,6 +16,8 @@ The code for the PX4 bootloader is available from the Github [Bootloader](https:
 ```bash
 git clone https://github.com/PX4/Bootloader.git
 cd Bootloader
+git submodule init
```

Comment:
Why not simpler invocation?

```bash
git clone https://github.com/PX4/Bootloader.git
cd Bootloader
git submodule update --init --recursive
```

Reply:
You can be even lazier.
`git clone --recursive https://github.com/PX4/Bootloader.git`

Reply:
I'm going to leave as is. Merging